dll/cpl/sysdm/general.c

```diff
@@ -598,73 +606,75 @@ static VOID GetSystemVersion(HWND hwnd)
     HeapFree(GetProcessHeap(), 0, pwszStr);
 }
 
-ULONGLONG GetSecondsQPC(VOID)
+static ULONGLONG GetTickCountQPC(VOID)
```

Comment:
Adding a comment telling in which units the tick-count is being returned, could be useful.

Reply:
Done.